WordPress/src/main/res/xml/stats_widget_info.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appwidget-provider
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:initialLayout="@layout/stats_widget_layout"
+    android:minHeight="40dp"
+    android:minWidth="310dp"
```

Comment:
The `minWidth` really should be a descriptive `dimens` value (ex: `STATS_WIDGET_FOUR_CELLS`).